include/ignition/math/Quaternion.hh

```diff
@@ -303,7 +303,7 @@ namespace ignition
       /// \deprecated Use SetFromAxisAngle(T, T, T, T)
       public: void IGN_DEPRECATED(7) Axis(T _ax, T _ay, T _az, T _aa)
       {
-        this->SetFromAxisAngle(_ax, _ay, _ax, _aa);
+        this->SetFromAxisAngle(_ax, _ay, _az, _aa);
```

Comment:
I get a warning about this (before this PR) with gcc 9. I looked in our CI and even though its running on Focal now, we still install and use gcc8.

Reply:
I think it's probably time to consider using the default on focal, rather than using the older gcc version.  We only needed gcc8 over gcc7.

Reply:
I agree. Default makes sense on focal.